docs/ingredients/accounts.md

```diff
@@ -0,0 +1,41 @@
+- Another type of account that can be used by a Program is a PDA
+- A PDA (Program derived Account) gives a program a way provide a signature
+
+#### Delivery
+Accounts store state (data and lamports). Lamports are fractional units of native SOL (0.000000001). While programs themselves do not store state directly, accounts can be used to store all the state a program requires. How does this work? Accounts when setup are assigned an owner. The owner is a program that has exclusive control over any changes to state. The owner program is referenced using a program id (address, or publickey). Anyone can read an account, but only the program that owns the account has the authority to change or permit modifications to the account data, including any debits or transfers from an account balance. If an account is able to recieve SOL or Tokens, anyone can transfer or send a balance to the account. If an account is not owned by a program, the program is only permitted to read its data and credit the account. Why might an account not be able to recieve SOL? 
+
+Accounts can be marked as executable. If the account is marked as executable it is treated as a program. Once marked executable the program data is immutable This is where a comparison to a file has been mentioned, as there appear to be some similarities to a file on your operating system. If a file is marked as executable, it can be loaded and executed by a program (in this case BPF Loader). If an account is not marked as executable, it is treated as a non executable account that can be read and updated based on permissions (authority and ownership), like a file. Accounts are marked executable during a successful program deployment by none other than the BPF Loader. The BPF loader is a native program of Solana and is in charge of deployments, upgrades, and executing programs on chain. This means the BPF Loader owns your program, and the program can own accounts in order to store any state it needs. Cool? 
+
+When creating transactions, accounts should be annotated as read-write or read-only accounts. Only set an account to 'writable (read-write)' if thet ransaction will make changes to the account. Marking an account as read-only enables parallel account processing between transactions. The Solana runtime permits read-only accounts to be read concurrently by multiple programs. If a program attempts to modify a read-only account, the transaction is rejected by the runtime. All Accounts to be written to or read must be passed to the program inside a transaction. If an on-chain program modifies a read-only account, the transaction will be reverted. The first account will always be read-write since it is used to cover transaction fees. This same account could be a wallet or keypair, and commongly referred to as the 'fee payer'. This fee payer would also be a signer on their account. More on signers coming up. Speaking of fees, accounts are charged rent. Rent?
```

Comment:
We should probably provide examples on read-write/read-only and explain why for that specific example. It would greatly help break up this section

Reply:
that goes more into an implementation example and should be what ingredients, recipes, guides etc are for.  As any transaction building will that these items, it should be example by implementation and actual use, instead of non applied theoretical examples. we could do code examples, but at that point it opens all the concepts up to having examples to demonstrate them all as well, and the concepts detailed within, which is what the other sections are for, to display the implementation and usage, like bf laid out in his get-program-account example with filters. here the concept of filters is described, and how to actually do it is shown in a guide/ingredient/recipe. let me know what you think.

Reply:
I think the core concepts should be more than just words on a page. Examples, breaks, etc help the reader digest what is on the page easier. Much like how Starry does his [docs](https://jstarry.notion.site/Transaction-Fees-f09387e6a8d84287aa16a34ecb58e239). Recipes are just that, references impls. Ingredients is a concept outside of the core building blocks. Core Concepts should be using everything in it's power to help explain to all audiences.